setup.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
+
 import provider
 
 setup(
-    name='django-oauth2-provider',
+    name='edx-django-oauth2-provider',
```

Comment:
I'm also wondering about how this name change will affect installations.  They had "django-oauth2-provider" installed, which provided a set of packages.  Presumably you'll be updating the requirements files to now install "edx-django-oauth2-provider", which will provide those same packages. How do we make sure that the new install will take precedence over the old?  Both will be installed.